physics/kepler_orbit_test.cpp

```diff
@@ -86,44 +260,1278 @@ TEST_F(KeplerOrbitTest, EarthMoon) {
-  EXPECT_THAT(*moon_orbit_from_state_vectors.elements_at_epoch().semimajor_axis,
-              AlmostEquals(*moon_orbit.elements_at_epoch().semimajor_axis, 1));
-  EXPECT_THAT(*moon_orbit_from_state_vectors.elements_at_epoch().mean_motion,
-              AlmostEquals(*moon_orbit.elements_at_epoch().mean_motion, 1));
-  EXPECT_THAT(moon_orbit_from_state_vectors.elements_at_epoch().inclination,
-              AlmostEquals(moon_orbit.elements_at_epoch().inclination, 1));
-  EXPECT_THAT(
-      moon_orbit_from_state_vectors.elements_at_epoch()
-          .longitude_of_ascending_node,
-      AlmostEquals(moon_orbit.elements_at_epoch().longitude_of_ascending_node,
-                   28));
-  EXPECT_THAT(
-      moon_orbit_from_state_vectors.elements_at_epoch().argument_of_periapsis,
-      AlmostEquals(moon_orbit.elements_at_epoch().argument_of_periapsis, 6));
-  EXPECT_THAT(moon_orbit_from_state_vectors.elements_at_epoch().mean_anomaly,
-              AlmostEquals(moon_orbit.elements_at_epoch().mean_anomaly, 6));
+              AlmostEquals(*MoonElements().mean_motion, 0));
+  EXPECT_THAT(*moon_orbit.elements_at_epoch().period,
+              AlmostEquals(*MoonElements().period, 0));
+  EXPECT_THAT(*moon_orbit.elements_at_epoch().periapsis_distance,
+              AlmostEquals(*MoonElements().periapsis_distance, 2));
+  EXPECT_THAT(*moon_orbit.elements_at_epoch().apoapsis_distance,
+              AlmostEquals(*MoonElements().apoapsis_distance, 1));
+  EXPECT_THAT(moon_orbit.elements_at_epoch().inclination,
+              AlmostEquals(MoonElements().inclination, 1));
+  EXPECT_THAT(moon_orbit.elements_at_epoch().longitude_of_ascending_node,
+              AlmostEquals(MoonElements().longitude_of_ascending_node, 28));
+  EXPECT_THAT(*moon_orbit.elements_at_epoch().argument_of_periapsis,
+              AlmostEquals(*MoonElements().argument_of_periapsis, 6));
+  EXPECT_THAT(*moon_orbit.elements_at_epoch().mean_anomaly,
+              AlmostEquals(*MoonElements().mean_anomaly, 6));
+  EXPECT_THAT(*moon_orbit.elements_at_epoch().true_anomaly,
+              AlmostEquals(*MoonElements().true_anomaly, 5));
+}
+
+TEST_F(KeplerOrbitTest, Voyager1) {
+  SolarSystem<ICRFJ2000Equator> solar_system(
+      SOLUTION_DIR / "astronomy" / "sol_gravity_model.proto.txt",
+      SOLUTION_DIR / "astronomy" /
+          "sol_initial_state_jd_2433282_500000000.proto.txt");
+  auto const sun = SolarSystem<ICRFJ2000Equator>::MakeMassiveBody(
+                         solar_system.gravity_model_message("Sun"));
+  MasslessBody const voyager1{};
+  constexpr Instant date = "2017-05-25T19:44:00,000"_TT;
+
+  Displacement<ICRFJ2000Equator> const expected_displacement(
+      {-4.202547896125371e+09 * Kilo(Metre),
+       -1.982453908150731e+10 * Kilo(Metre),
+        4.378406169173994e+09 * Kilo(Metre)});
+  Velocity<ICRFJ2000Equator> const expected_velocity(
+      {-2.067668772297011e+00 * (Kilo(Metre) / Second),
```

Comment:
We should decide which one we want... https://github.com/mockingbirdnest/Principia/pull/809#r49282153